src/main/java/pl/javastart/task/TimePatterns.java

```diff
@@ -0,0 +1,21 @@
+package pl.javastart.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class TimePatterns {
+    public static LocalDateTime parseTimeFormat(String userInput) {
+        DateTimeFormatter dateFormatterWithTime = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+        DateTimeFormatter dateFormatterAlternative = DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm:ss");
+        LocalDateTime parsedDateTime = null;
+        if (userInput.contains(" ") && userInput.contains("-")) {
+            parsedDateTime = LocalDateTime.parse(userInput, dateFormatterWithTime);
+        } else if (userInput.contains(".")) {
+            parsedDateTime = LocalDateTime.parse(userInput, dateFormatterAlternative);
+        } else {
+            userInput += " 00:00:00";
+            parsedDateTime = LocalDateTime.parse(userInput, dateFormatterWithTime);
+        }
+        return parsedDateTime;
```

Comment:
Tutaj trochę zhackowane podejście. Jeśli doszedłby kolejny format z `-` to byłby trochę problem. Lepiej gdybyś zrobił listę z dostępnymi formatami. Jedna lista mogłaby zawierać formaty z datą i czasem, a druga tylko z datą. W pętli próbuj formatować jeden za drugim i wyłapuj wyjątki i je ignoruj. Pierwszą datę, którą uda się poprawnie sformatować możesz zwrócić jako wynik metody.